File: Modul-3/Soal-4/PRAK304-2410817110016-MuhammadAzriRahman.c
#include <stdio.h>

void main() {
    int a;

    printf("Masukan Angka antara 0-99:\0");
    scanf(" %d", &a);

    if (a > 99) {
        printf("Anda Menginput Melebihi Limit Bilangan\n");
    } else if (a == 0) {
        printf("Nol");
    } else if (a > 0 && a < 10) {
        printf("Satuan");
    } else if (a == 10 || a > 19) {
        printf("Puluhan");
    } else {
        printf("Belasan");
    }
}

File: Modul-3/Soal-4/PRAK304-2410817110016-MuhammadAzriRahman.py
a = int(input("Masukan Angka antara 0-99: "))

if (a > 99):
    print("Anda Menginput Melebihi Limit Bilangan")
elif (a == 0):
    print("Nol")
elif (a > 0 and a < 10):
    print("Satuan")
elif (a == 10 or a > 19):
    print("Puluhan")
else:
    print("Belasan")

